services/core-api/app/api/now_applications/resources/now_application_document_type_resource.py

```diff
@@ -73,7 +73,9 @@ def post(self, document_type_code):
             NOW_DOCUMENT_DOWNLOAD_TOKEN(token), {
                 'document_type_code': document_type_code,
                 'now_application_guid': data['now_application_guid'],
-                'template_data': template_data
+                'template_data': template_data,
+                'username': User().get_user_username(),
+                'authorization_header': request.headers['Authorization']
```

Comment:
For now, we don't have a "proper" means of authorizing communication between our microservices, so this temporary solution has been put in place to authorize with the document manager (pass the authorization headers into the token and re-use them later). A ticket to set something else up as been created [here](https://bcmines.atlassian.net/browse/MDS-2744?atlOrigin=eyJpIjoiM2NiYTk2N2YyNTkwNDFiMGE2ODYwNWVlOTZiNDI4YTAiLCJwIjoiaiJ9).

Reply:
Are you able to put this comment into the code as well in the form of a note or `TODO`?

Reply:
Also, this resource will be where we are enabling the 'preview' functionality as well, right? We'll probably want to implement the service account ticket before doing the immediate preview.

Reply:
Added comment